recipes/rocksdb/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "rocksdb" %}
+{% set version = "6.2.2" %}
+
+package:
+  name: rocksdb
+  version: {{ version }}
+
+source:
+  url: https://github.com/facebook/rocksdb/archive/v{{ version }}.tar.gz
+  sha256: 3e7365cb2a35982e95e5e5dd0b3352dc78573193dafca02788572318c38483fb
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    - {{ pin_subpackage(name, max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler("cxx") }}
+    - cmake >=3.5.1,<4
+
+  host:
+    - gflags >=2.2.2,<3
+    - jemalloc >=4.5.0,<5
+    - snappy >=1.1.7,<2
+  run:
+    - {{ pin_compatible("jemalloc", exact=True) }}
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/librocksdb.a
+    - test -f ${PREFIX}/lib/librocksdb.dylib  # [osx]
+    - test -f ${PREFIX}/lib/librocksdb.so     # [linux]
+
+about:
+  home: https://rocksdb.org
+  license: Apache-2.0
+  license_file: LICENSE.Apache
+  summary: A library that provides an embeddable, persistent key-value store for fast storage.
+
+  dev_url: https://github.com/facebook/rocksdb
+  doc_url: https://rocksdb.org/docs
+
+extra:
+  recipe-maintainers:
+    - Yangqing
+    - sodre
```

```suggestion
    - benhuff
    - sodre
```
@benhuff, do you agree to be a maintainer?

---

We need @oblute and @rluria14 to comment that they are ok maintaining here.

---

I think they already had... it must have dropped off when I force pushed...

---

@beckermr,
I mistakenly asked them to approve the change while developing the cockroachdbPR, see it here: https://github.com/conda-forge/staged-recipes/pull/11307#discussion_r409654359

---

i agree to be a maintainer

---

I also agree to be a maintainer

---

@conda-forge/staged-recipes, all maintainers have agreed, and the reviews comments were handled. This is ready to merge.